blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/AbstractPipelineCreateRequest.java

```diff
@@ -41,6 +48,6 @@
         if(!acl.hasCreatePermission(a, p, descriptor)){
             throw new ServiceException.ForbiddenException("Missing permission: " +Item.CREATE.group.title+"/"+Item.CREATE.name + Item.CREATE + "/" + descriptor.getDisplayName());
         }
-        return parent.createProject(descriptor, name, true);
+        return Jenkins.getInstance().createProject(descriptor, name, true);
```

Comment:
`parent.createProject()` is fine, at the abstract level, better for client to decide where to create this project.